pkg/minikube/firewall/firewall.go

```diff
@@ -75,6 +76,7 @@ func UnblockBootpd() error {
 			klog.Infof("%v may require a password: %v", c.Args, err)
 			if !viper.GetBool("interactive") {
 				klog.Warningf("%s requires a password, and --interactive=false", c.Args)
+				c.Args = slices.Insert(c.Args, 1, "-n")
```

Comment:
thanks for limitting it only for "interactive" to avoid  Unintended Privilege Escalation like if the user has recently used sudo for a different purpose. This could pose a security risk if the script is compromised or contains errors.
but it makes sense for not interactive